docs/guides/03-Styling-your-visualization-using-Expressions.md

```diff
@@ -0,0 +1,198 @@
+
+Once you have a `Visualization` object, you can:
+- Get its property values by using the `.value` getter: `const currentColor = viz.color.value;`
+- Modify its property values by using the`.blendTo()` method: `viz.color.blendTo(‘green’);`
+
+<div class="example-map">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/viz-properties/accessing-viz-properties.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-color-spaces">View my source code!</a>
+</div>
+
+
+### The String and the JS Visualization APIs
+
+CARTO VL provides two equivalent and similar APIs to define visualizations. Until now, we have seen the String API which is nothing more than a small layer of syntax sugar for the JS API. In fact, internally, the usage of the String API is implemented through the usage of the public JS API.
+
+Usage of the JS API can facilitate some advanced topics, but both are equivalent and everything that can be done through the JS API can be done with the String API and vice versa.
+
+In the reference documentation you’ll be able to see examples with both APIs for each expression, but let’s see some examples:
+
+```Javascript
+// Usage of the String API
+const viz = new carto.Viz(`
+    width: 5
+    color: red
+`);
+```
+
+```Javascript
+// Usage of the JS API
+const s = carto.expressions;
+const viz = new carto.Viz({
+    width: 5
+    color: s.namedColor(‘red’)
+});
+```
+
+As you can see, the String API provides some syntax sugar for named colors. It also provides easier usage of:
+Arithmetic operators: `+, `/`, `^`...
+Boolean operators: `and`, `or`, `not`...
+Eliminates the need to access the `carto.expressions` namespace
+`#FFF` like color constants
+Accessing data properties with `$propertyName`
+Variables
+
+### Variables
+
+As we’ve seen, CARTO VL visualization language follows the next pattern: `property: expression`, where expression can be a constant or a function call of the form `functionName(param1, param2, …)` or a built-in infix function call like `param1/param2`.
+
+The usage of variables is indeed not required for applying any of the CARTO VL styling capabilities. However, it is required for [Interactivity] and it can be used to simplify some visualizations.
+
+We’ll cover this topic in depth on the [Interactivity] guide.
+
+### Comparison to CSS-derived languages
+
+In contrast with cascading languages like CSS and CSS-derived languages, CARTO VL styling properties cannot be redefined based on selectors. Instead, CARTO VL expressions like `ramp` are used to select the expected values based on some input (the selector).
+
+This is a comparison between both approaches to get the gist of it, it's ok if you don't fully understand them since we'll cover this in depth later in the data-driven visualization guide.
```

Comment:
if you don't fully understand them now...

Reply:
The example uses `ramp` which is not covered until later, but it's difficult to explain the "CARTO VL" way compared to the CartoCSS way without using it

Reply:
Yeah, I was just adding the word "now"... 😄

Reply:
ah, I see, :+1: